src/main/java/frc/robot/subsystems/internals/InternalSubsystem.java

```diff
@@ -47,6 +47,7 @@
     private int entranceStorageBallCount = 0;
     private int storageStagingBallCount = 0;
     private int stagingExitBallCount = 0;
+    private int totalBallCount = 0;
```

Comment:
Shouldn't `totalBallCount` be the sum of `entranceStorageBallCount`, `storageStagingBallCount`, and `stagingExitBallCount` or are those fields not used anymore?